package.json

```diff
@@ -9,6 +9,8 @@
         "lint": "next lint"
     },
     "dependencies": {
+        "@0xsquid/sdk": "^2.8.24",
+        "@0xsquid/squid-types": "^0.1.104",
```

Comment:
why is this necessary? for x-chain claiming we didn't need it, why do we need it now?

Reply:
Please lets be consistent with how we do xchain claiming and xchain request payment